### PR TITLE
remove_tags_109

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Description
This PR removes non used tags/use_cases in our approved_tags/use_cases.json files, to further lower the possibility of abusive usage of the list.

more PRs that will trim the list will follow.

I verified that each use-case is not being used in the repo - please double check me.

related PRs:
https://github.com/demisto/content/pull/23898

### CODE REVIEW

**Review by ShahafBenYakir — COMMENTED** [review]
Let's run nightly test to make sure validation will not fail after removing the tags and use cases

**Comment by bakatzir** [comment]
> Let's run nightly test to make sure validation will not fail after removing the tags and use cases
Correct, triggered - https://code.pan.run/xsoar/content/-/pipelines/4457697

File: Packs/Arcanna/pack_metadata.json [hunk 1]
```diff
@@ -10,10 +10,7 @@
     "categories": [
         "Data Enrichment & Threat Intelligence"
     ],
-    "tags": [
-        "Security",
-        "Machine Learning"
-    ],
```
Comment (marian-craciunescu) [inline Packs/Arcanna/pack_metadata.json:16]: Here it should be "Decision Intelligence"

**Review by marian-craciunescu — CHANGES_REQUESTED** [review]

File: Packs/Arcanna/pack_metadata.json (hunk 1 above)
Comment (bakatzir) [reply Packs/Arcanna/pack_metadata.json:16]: Hey Marian,
we are reviewing and updating the list of approved tags, your Partner Success Engineer will contact you regarding the possibilities in the near future. First, we are aligning the XSOAR and XSIAM Marketplaces to the approved baseline.

**Comment by bakatzir** [comment]
@ShahafBenYakir validations passed, but xsiam server GA failed. does not seem related, your call.

**Review by ShahafBenYakir — APPROVED** [review]
looks good to me

**Comment by bakatzir** [comment]
@dorschw  / @yaakovpraisler would love your opinion as well prior to merging.

**Review by yaakovpraisler — APPROVED** [review]
Looks good

**Pull request merged**